components/ButtonBar/src/MDCButtonBar.m

```diff
@@ -476,7 +476,7 @@ - (void)setButtonsTitleColor:(nullable UIColor *)color forState:(UIControlState)
   [_defaultBuilder setTitleColor:color forState:state];
 
   for (UIView *viewObj in _buttonViews) {
-    if ([viewObj isKindOfClass:[UIButton class]]) {
+    if ([viewObj isKindOfClass:[MDCButton class]]) {
       MDCButton *button = (MDCButton *)viewObj;
       [button setTitleColor:color forState:state];
```

Comment:
Possibly similarly here.

Reply:
This could be UIButton, I think either cast would be fine but I'm going to go with MDCButton for consistency.